contrib/test/integration/e2e-base.yml

```diff
@@ -1,4 +1,18 @@
 ---
+# only fixup paths for e2e tests which expect to be able to pass 'test-handler' as the runtime handler
+- name: fixup default runtime for k8s suite
+  replace:
+    regexp: 'default_runtime = "runc"'
```

Comment:
Instead of replacing runc, we should add another runtime. However, let the current e2e runs verify the fix first :)

Reply:
fix incoming once e2e_fedora also (hopefully) goes green